### PR TITLE
Bulk added R recipes for migration from bioconda (5)

### PR DESCRIPTION
**Pull request by jenzopr**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

See also https://github.com/bioconda/bioconda-recipes/issues/13964 

All packages are `noarch: generic`.

### CODE REVIEW

**Comment by jenzopr** [comment]
`r-mmod` does not build for Win platforms because dependencies (`r-pegas` and `r-adegenet`) are not available. Skip build here or in the feedstock? Ping @dbast

**Comment by dbast** [comment]
@jenzopr As those all are 100% noarch recipes and as only some builds fail on not-linux due to dependencies missing, nothing to skip here on PR nor on feedstock side... the PR could be already merged... I was just waiting a bit before merging this, to see all the previous packages to be properly triggered in the right order and uploaded :)

File: recipes/r-metstat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - r-pls
+
+test:
+  commands:
+    - $R -e "library('MetStaT')"           # [not win]
+    - "\"%R%\" -e \"library('MetStaT')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=MetStaT
+  license: Apache (== 2.0)
```
Comment (dbast) [inline recipes/r-metstat/meta.yaml]: The most canonical string here would be the short identifier according to spdx: `Apache-2.0` ... see https://spdx.org/licenses/Apache-2.0.html ... just to consider for the future ... we don't have this consistent in all the other r feedstocks...

Comment (jenzopr) [reply recipes/r-metstat/meta.yaml]: Thanks!

**Comment by dbast** [comment]
Retrigger exercise done :)

**Pull request merged**